frontend/src/components/elements/Button/Button.js

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import PropTypes from 'prop-types';
+import ButtonStyled from './Button.styled';
+
+export default function Button({ children, onClick, ...props }) {
+
+  const handleMouseUp = e => onClick(e);
+
+  return <ButtonStyled {...props} onMouseUp={handleMouseUp} >{children}</ButtonStyled>;
+}
+
+/* Props */
+export const POSITIVE = "positive";
+export const CAUTION = "caution";
+export const NEUTRAL = "neutral";
```

Comment:
Can we create a `constants.js` file that contains all of our constants on the frontend. We eventually may also want to have `utils.js` file that has all of our utility functions.

Reply:
So can we import `POSITIVE`, `CAUTION`, and `NEUTRAL` from that file.

Reply:
I think we'll probably end up with a constants file, or several of them. But I like the idea of keeping constants scoped. The nice thing about the way this is done here is you can do this:
`import Button, { POSITIVE, CAUTION, NEUTRAL } from "./Button/Button"`

It makes it clear to the user what those states are for and how they should be used.